### PR TITLE
refactor: move history hive tests to history folder add instructions for where to put simulators

### PR DESCRIPTION
**Pull request by KolbyML**
These PR's should be reviewed/merged in this order
1. https://github.com/ethereum/portal-hive/pull/118
2. https://github.com/ethereum/portal-hive/pull/124
3. https://github.com/ethereum/portal-hive/pull/125
4. https://github.com/ethereum/portal-hive/pull/126
5. https://github.com/ethereum/portal-hive/pull/127

This is the first PR to add new simulators/tests for different networks like beacon.

I will do a series of PR's built off each other so reviewing/merges can be done quick and seemless.

None of these PR's should require a deep review the goal is just to move things around to make it possible to add these new simulators/tests

### CODE REVIEW

File: simulators/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Portal Simulators
+Portal simulators are sorted by folders indictacting what networks are being tested.
```
Comment (njgheorghita) [inline simulators/README.md:2]: `indicating`

File: simulators/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+The names of folder are
+- all lowercase
+- dash seperated
```
Comment (njgheorghita) [inline simulators/README.md:6]: `separated`

File: simulators/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+## Examples of valid folder names :heavy_check_mark: 
+- ``history``
+- ``beacon``
+- ``state``
+- ``beacon-history``
+- ``beacon-history-state``
+- ``beacon-state``
+- ``history-state``
+
+## Examples of invlaid folder names :x:
```
Comment (njgheorghita) [inline simulators/README.md:18]: `invalid`

File: .circleci/config.yml [hunk 4]
```diff
@@ -60,15 +60,15 @@ jobs:
   build-simulator-dockerfile:
     docker:
       - image: cimg/base:2022.06
-    parallelism: 5
+    parallelism: 6
```
Comment (njgheorghita) [inline .circleci/config.yml:63]: It's probably worth leaving a comment as to why the increased parallelism was necessary

**Review by njgheorghita — APPROVED** [review]
:shipit:

**Review by ogenev — APPROVED** [review]

**Pull request merged**